transport/grpc/interceptor.go

```diff
@@ -76,7 +79,18 @@ func (s *Server) streamServerInterceptor() grpc.StreamServerInterceptor {
 			replyHeader: headerCarrier(replyHeader),
 		})
 
-		ws := NewWrappedStream(ctx, ss)
+		h := func(ctx context.Context, req interface{}) (interface{}, error) {
+			ctx = context.WithValue(ctx, stream{
+				ServerStream: ss,
+				middlware:    s.middleware,
+			}, ss)
+			return handler(srv, ss), nil
+		}
+		if next := s.middleware.Match(info.FullMethod); len(next) > 0 {
```

Comment:
Additionally, I think we should add a configuration item to specify which middleware is used for streaming send recv

Reply:
What would you recommend?

Reply:
grpc.StreamMiddleware

Reply:
Do you mean `grpc.ServerStream`? There is no `grpc.StreamMiddleware`